pkg/sql2pgroll/drop.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package sql2pgroll
+
+import (
+	pgq "github.com/pganalyze/pg_query_go/v6"
+
+	"github.com/xataio/pgroll/pkg/migrations"
+)
+
+// convertDropStatement converts supported drop statements to pgroll operations
+func convertDropStatement(stmt *pgq.DropStmt) (migrations.Operations, error) {
+	switch stmt.RemoveType {
+	case pgq.ObjectType_OBJECT_INDEX:
+		return convertDropIndexStatement(stmt)
+	}
+	return nil, nil
+}
+
+// convertDropIndexStatement converts simple DROP INDEX statements to pgroll operations
+func convertDropIndexStatement(stmt *pgq.DropStmt) (migrations.Operations, error) {
+	if !canConvertDropIndex(stmt) {
+		return nil, nil
+	}
+	s := stmt.Objects[0].GetList().Items[0].GetString_().GetSval()
+	return migrations.Operations{
+		&migrations.OpDropIndex{
+			Name: s,
+		},
+	}, nil
+}
+
+// canConvertDropIndex checks whether we can convert the statement without losing any information.
+func canConvertDropIndex(stmt *pgq.DropStmt) bool {
+	if len(stmt.Objects) > 1 {
+		return false
+	}
+	if stmt.MissingOk || stmt.Concurrent {
```

Comment:
`pgroll` always drops indexes concurrently (see `op_drop_index.go`).

Conversion should succeed whether the `CONCURRENTLY` option is set or not.

Reply:
I see we also use `IF EXISTS`

Reply:
yes, I think we should convert regardless of whether `IF EXISTS` is specified or not.